### PR TITLE
Fix installation process

### PR DESCRIPTION
**Pull request by Erwyn**
* Change password generation function
* Do not attempt to move /home/codendiadm, /home/users, /home/groups
* Do not attempt to write libnss-mysql configuration

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -26,7 +26,6 @@ fi
 /usr/share/tuleap/tools/docker/tuleap-aio/fix-owners.sh
 
 # Update DB location
-sed -i "s/^host.*/host db/" /etc/libnss-mysql.cfg
```
Comment (LeSuisse) [inline run.sh:29]: We need to keep this one for instance having migrated from older stacks (or for people doing the setup manually to troubleshoot issues of production instances).

Note that is not that much of an issue because this script is not strict with error (i.e. there is no `set -e`) but adding a simple guard like `[ -f /etc/libnss-mysql.cfg ] && ` would not hurt.

File: boot-install.sh [hunk 2]
```diff
@@ -3,7 +3,8 @@
 set -ex
 
 function generate_passwd {
-   cat /dev/urandom | tr -dc "a-zA-Z0-9" | fold -w 15 | head -1
+    (tr -dc 'a-zA-Z0-9' < /dev/urandom | head -c32) 2>/dev/null
+    printf ""
```
Comment (LeSuisse) [inline boot-install.sh:7]: Can you please explain why this change is needed to fix the installation process?

File: run.sh (hunk 1 above)
Comment (Erwyn) [reply run.sh:29]: done

File: boot-install.sh (hunk 2 above)
Comment (Erwyn) [reply boot-install.sh:7]: There is an issue with the current password generation that get stuck.

I took the one in the setup.el7 process.
